notebooks/systemds_dev.ipynb

```diff
@@ -0,0 +1,882 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "SystemDS on Colaboratory.ipynb",
+      "provenance": [],
+      "collapsed_sections": [],
+      "include_colab_link": true
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "view-in-github",
+        "colab_type": "text"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github.com/apache/systemds/blob/master/notebooks/colab/systemds_dev_standalone.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "_BbCdLjRoy2A",
+        "colab_type": "text"
+      },
+      "source": [
+        "## Developer notebook for Apache SystemDS"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "I76oWp7foiyF",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        ""
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
```

Comment:
This above code cell contains nothing, Maybe make some introduction text saying ...

```code
In this notebook we show ... x ... becauase ... y etc. 
it is separated into 3 parts...
1. install [link]
2. code [link]
3. results [link]
```

Reply:
I have reorganized the sections with more description!